data/mods/DinoMod/mapgen/microlab_DinoLab.json

```diff
@@ -0,0 +1,1150 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "microlab_DinoLab_sub_entry" ],
+    "weight": 1000,
+    "object": {
+      "fill_ter": "t_concrete",
+      "rows": [
+        "|||||||||||  |||||||||||",
+        " cc  c    |  |^BBB|     ",
+        " cc  c   c|  |   h|r i  ",
+        "         c|  |  dd|r c  ",
+        " FFFFccccc|  |   h|r c  ",
+        "|||||||||||YY|[===||    ",
+        " FF c|^???^       v| c  ",
```

Comment:
What is `v` supposed to be here? Looks like either a typo or something defined in C:BN.